### PR TITLE
all: set map len

### LINKED ISSUE #188: Set map length where it possible

### PR DESCRIPTION
**Pull request by sashamelentyev**
Closes #188

### CODE REVIEW

File: cfg/config.go [hunk 1]
```diff
@@ -389,19 +389,21 @@ func ParseField(v reflect.Value, vField reflect.Value, tField reflect.StructFiel
 
 			finalField.SetInt(int64(result))
 		case "list-map":
-			listMap := make(map[string]bool)
-
 			parts := strings.Split(vField.String(), ",")
+
```
Comment (ansakharov) [inline cfg/config.go:393]: Remove blank line here

File: cfg/config.go [hunk 2]
```diff
@@ -389,19 +389,21 @@ func ParseField(v reflect.Value, vField reflect.Value, tField reflect.StructFiel
...
 				cleanPart := strings.TrimSpace(part)
 				listMap[cleanPart] = true
 			}
 
 			finalField.Set(reflect.ValueOf(listMap))
 		case "list":
-			list := make([]string, 0)
-
 			parts := strings.Split(vField.String(), ",")
+
```
Comment (ansakharov) [inline cfg/config.go:404]: Remove blank line here

**Review by ansakharov — CHANGES_REQUESTED** [review]
Please add issue according to: https://github.com/ozontech/file.d/blob/master/CONTRIBUTING.md

**Comment by sashamelentyev** [comment]
@ansakharov thank you for review!
All done.

**Review by ansakharov — APPROVED** [review]
LGTM.
But issue is empty :(

**Pull request merged**